lib/kube/proxy/portforward_spdy.go

```diff
@@ -326,8 +309,8 @@ func (h *portForwardProxy) run() {
 				go h.monitorStreamPair(p, time.After(h.streamCreationTimeout))
 			}
 			if complete, err := p.add(stream); err != nil {
-				msg := fmt.Sprintf("error processing stream for request %s: %v", requestID, err)
-				p.printError(msg)
+				err := trace.Errorf("error processing stream for request %s: %v", requestID, err)
```

Comment:
I've been told before to avoid naked trace.Errorf, so passing it along:

```suggestion
				err := trace.BadParameter("error processing stream for request %s: %v", requestID, err)
```